lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions_controller.rb

```diff
@@ -30,7 +30,36 @@ def edit
     content_block_edition = ContentObjectStore::ContentBlock::Edition.find(params[:id])
     @schema = ContentObjectStore::ContentBlock::Schema.find_by_block_type(content_block_edition.document.block_type)
 
-    @form = ContentObjectStore::ContentBlock::EditionForm::Update.new(content_block_edition:, schema: @schema)
+    @form = ContentObjectStore::ContentBlock::EditionForm::Update.new(
+      content_block_edition:, schema: @schema, edition_to_update_id: content_block_edition.id,
+    )
+  end
+
+  def review_links
```

Comment:
as discussed with @pezholio and @tahb  - having a separate controller action per page of the form is not sustainable in long term, but to get an initial PR over the line this is what I am suggesting.